### DISCUSSION TITLE
Temas pendientes

### DISCUSSION THREAD

**Discussion by mchalin** [CodeSystem2022/UTN_BsAs_Tercer_Semestre#47] [General]
Gente, buenos dias! Solo para dejar acentado cositas pendientes a cerrar estos días. 

INGLES: Tenemos que grabar la entrevista laboral. 
LABORATORIO: Python, proyecto final (POO, herencia, polimorfismo, try catch, base de datos) Definir si hacemos el mismo proyecto o uno nuevo. 
PROGRAMACION: Generar un PDF o video con el paso a paso de una semana de trabajo. Yo me ofrezco en la semana del 29/5 al 4/6 a hacerlo a medida q va avanzando la semana. Sino, abierto a cualquiera q quiera hacerlo

**Comment by alealgarin92** [comment]
Grupos para entrevista de ingles: 

1-Vivi/Maxi
2-Dario/Fer
3-Pia/Ale

**Reply by mchalin** [reply]
Hi @vmgith ! How are u ?

**Reply by vmgith** [reply]
hello, getting ready to interview you

**Reply by piaolivera** [reply]
Hola. agarre la primera entrevista que envió @DarioVega en wsp para empezar a practicarla!!
